includes/class-stats.php

```diff
@@ -3154,4 +3156,38 @@ private function set_date_ranges() {
 			$this->relative_date_ranges[ $range ] = $dates;
 		}
 	}
+
+	/**
+	 * Based on the query_vars['revenue_type'], use gross or net statuses.
+	 *
+	 * @since 3.0
+	 *
+	 * @return array The statuses of orders to use for the stats generation.
+	 */
+	private function get_revenue_type_statuses() {
+
+		$statuses = edd_get_gross_order_statuses();
+		if ( 'net' === $this->query_vars['revenue_type'] ) {
+			$statuses = edd_get_net_order_statuses();
```

Comment:
If this is changed to a `return` statement, then line 3169 can be removed and the final return can just return gross statuses--no need to get them otherwise, and since there's not a filter, we can do an early return.